shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -54,6 +55,7 @@ @implementation FlutterEngine {
   flutter::ThreadHost _threadHost;
   std::unique_ptr<flutter::Shell> _shell;
   NSString* _labelPrefix;
+  NSString* _initialRoute;
```

Comment:
This is leaking, please use a property with `copy` semantics.

Reply:
The releasing of it is conditional.  There is still the opportunity for it to leak.  It should be cleaned up in the dealloc.

We are copying it, using the property syntax just makes the memory management technique apparent.

Reply:
Ah understood. PTAL

Reply:
lookin good